plugins/node/opentelemetry-instrumentation-connect/README.md

```diff
@@ -0,0 +1,70 @@
+# OpenTelemetry Connect Instrumentation for Node.js
+
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+This module provides automatic instrumentation for [`connect`](https://github.com/senchalabs/connect).
+
+For automatic instrumentation see the
+[@opentelemetry/node](https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-node) package.
+
+## Installation
+
+This instrumentation relies on HTTP calls to also be instrumented. Make sure you install and enable both, otherwise you will not see any spans being exported from the instrumentation.
```

Comment:
Does it really absolutely require http instrumentation? I don't see that in the current implementation.

Reply:
updated description basically it will export spans which will have no common parent, http is quite important here the same as in express